src/backend/distributed/metadata/metadata_utility.c

```diff
@@ -635,6 +662,39 @@ GenerateSizeQueryOnMultiplePlacements(List *shardIntervalList,
 }
 
 
+/*
+ * GetWorkerPartitionedSizeUDFNameBySizeQueryType returns the corresponding worker
+ * partitioned size query for given query type.
+ * Errors out for an invalid query type.
+ */
+static char *
+GetWorkerPartitionedSizeUDFNameBySizeQueryType(SizeQueryType sizeQueryType)
+{
+	switch (sizeQueryType)
+	{
+		case RELATION_SIZE:
```

Comment:
can we add test to here as the code-cov shows we don't cover? also for `TOTAL_RELATION_SIZE`

Reply:
This is also not coverable because `citus_shard_cost_by_disk_size` calls here only with the type `TOTAL_RELATION_SIZE`. The other `worker_partitioned_*_size` functions are not used in the code at the moment.